From: SVN-Git Migration <python-modules-team@lists.alioth.debian.org>
Date: Thu, 8 Oct 2015 10:17:29 -0700
Subject: local_inventory

Use local copies of object.inv for building documentation.
Doesn't need forwarding.
Patch by Scott Kitterman <scott@kitterman.com> Feb 2, 2012
Patch-Name: local_inventory
---
 doc/src/conf.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/src/conf.py b/doc/src/conf.py
index 6afb286..3af0b3e 100644
--- a/doc/src/conf.py
+++ b/doc/src/conf.py
@@ -61,8 +61,8 @@ except ImportError:
     release = version
 
 intersphinx_mapping = {
-    'py': ('http://docs.python.org/2', None),
-    'py3': ('http://docs.python.org/3', None),
+    'py': ('/usr/share/doc/python-doc/html', None),
+    'py3': ('/usr/share/doc/python3-doc/html', None),
     }
 
 # Pattern to generate links to the bug tracker
